Library/Formula/gnu-units.rb

```diff
@@ -28,6 +28,11 @@ class GnuUnits < Formula
   patch :DATA
 
   def install
+    # OSX doesn't provide 'python2', but on some Linux distributions
```

Comment:
We can shrink these notes a bit. Something like `OS X does not provide a `python2` executable` or something. It's a relatively well known issue that doesn't need too much explaining.